src/omero_zarr/masks.py

```
@@ -272,14 +272,15 @@ def masks_to_labels(
                 labels.shape, mask_shape
             )
 
-        fillColors = {}
+        fillColors = [None]
         for count, shapes in enumerate(masks):
             # All shapes same color for each ROI
             print(count)
+            maskColor = None
             for mask in shapes:
                 # Unused metadata: the{ZTC}, x, y, width, height, textValue
-                if mask.fillColor:
-                    fillColors[count + 1] = unwrap(mask.fillColor)
+                if mask.fillColor and maskColor is None:
+                    maskColor = unwrap(mask.fillColor)
```

Should we throw an exception is masks in the same ROI have different colours?